tests/test_utils.py

```diff
@@ -40,6 +39,18 @@ def test_get_image_location_ok(tmpdir):
     )
 
 
+def test_get_image_location_unicode_path_ok(tmpdir):
```

Comment:
can you also add a test where you put a file path that is **not** valid UTF-8?

Reply:
Done :+1: